processing/src/main/java/io/druid/query/topn/AlphaNumericTopNMetricSpec.java

```diff
@@ -46,7 +46,7 @@ public AlphaNumericTopNMetricSpec(
   @Override
   public Comparator getComparator(List<AggregatorFactory> aggregatorSpecs, List<PostAggregator> postAggregatorSpecs)
   {
-    return new AlphanumComparator();
+    return ComparatorUtils.NATURAL_COMPARATOR;
```

Comment:
the code is LGPL, so it should be fine to include http://www.davekoelle.com/alphanum.html
No reason to remove LGPL dependencies.